src/frontend/src/tests/lib/utils/onramper.utils.spec.ts

```diff
@@ -7,263 +7,210 @@ import {
-			{ cryptoId: ICP_TOKEN.buy?.onramperId ?? '', wallet: mockAccountIdentifierText }
-		];
+			const expectedWallets: OnramperNetworkWallet[] = [
+				{ networkId: ICP_NETWORK.buy?.onramperId ?? 'icp', wallet: mockAccountIdentifierText },
+				{ networkId: ETHEREUM_NETWORK.buy?.onramperId ?? 'icp', wallet: mockEthAddress }
+			];
 
-		const result = mapOnramperWallets({ tokens, walletMap: newWalletMap });
+			const result = mapOnramperNetworkWallets({ networks, walletMap });
 
-		expect(result).toEqual(expectedWallets);
-	});
-});
+			expect(result).toEqual(expectedWallets);
+		});
 
-describe('mapOnramperNetworkWallets', () => {
-	const walletMap: Map<symbol, Option<OnramperWalletAddress>> = new Map([
-		[BTC_MAINNET_NETWORK_ID, mockBtcAddress],
-		[ETHEREUM_NETWORK_ID, mockEthAddress],
-		[ICP_NETWORK_ID, mockAccountIdentifierText]
-	]);
+		it('should return an empty array if no valid tokens are present', () => {
+			const networks = [SEPOLIA_NETWORK];
 
-	it('should return correct wallets when tokens have valid cryptoIds and wallets', () => {
-		const networks = [ICP_NETWORK, ETHEREUM_NETWORK, SEPOLIA_NETWORK];
+			const result = mapOnramperNetworkWallets({ networks, walletMap });
 
-		const expectedWallets: OnramperNetworkWallet[] = [
-			{ networkId: ICP_NETWORK.buy?.onramperId ?? 'icp', wallet: mockAccountIdentifierText },
-			{ networkId: ETHEREUM_NETWORK.buy?.onramperId ?? 'icp', wallet: mockEthAddress }
-		];
-
-		const result = mapOnramperNetworkWallets({ networks, walletMap });
-
-		expect(result).toEqual(expectedWallets);
-	});
-
-	it('should return an empty array if no valid tokens are present', () => {
-		const networks = [SEPOLIA_NETWORK];
-
-		const result = mapOnramperNetworkWallets({ networks, walletMap });
-
-		expect(result).toEqual([]);
-	});
+			expect(result).toEqual([]);
+		});
 
-	it('should handle cases where wallet addresses are null or undefined', () => {
-		const networks = [ICP_NETWORK, ETHEREUM_NETWORK];
+		it('should handle cases where wallet addresses are null or undefined', () => {
+			const networks = [ICP_NETWORK, ETHEREUM_NETWORK];
 
-		const newWalletMap: Map<symbol, Option<OnramperWalletAddress>> = new Map(walletMap);
-		newWalletMap.set(ETHEREUM_NETWORK_ID, undefined);
+			const newWalletMap: Map<symbol, Option<OnramperWalletAddress>> = new Map(walletMap);
+			newWalletMap.set(ETHEREUM_NETWORK_ID, undefined);
 
-		const expectedWallets: OnramperNetworkWallet[] = [
-			{ networkId: ICP_NETWORK.buy?.onramperId ?? 'icp', wallet: mockAccountIdentifierText }
-		];
+			const expectedWallets: OnramperNetworkWallet[] = [
+				{ networkId: ICP_NETWORK.buy?.onramperId ?? 'icp', wallet: mockAccountIdentifierText }
+			];
 
-		const result = mapOnramperNetworkWallets({ networks, walletMap: newWalletMap });
+			const result = mapOnramperNetworkWallets({ networks, walletMap: newWalletMap });
 
-		expect(result).toEqual(expectedWallets);
+			expect(result).toEqual(expectedWallets);
+		});
```

Comment:
here I just removed the tests for `mapOnramperWallets` and wrapped everything around a single `describe`